Tofu/AccountCell.swift

```diff
@@ -65,7 +69,6 @@ private func imageForAccount(_ account: Account) -> UIImage {
     case .some("Tumblr"): return UIImage(named: "Tumblr")!
     case .some("Ubisoft"): return UIImage(named: "Ubisoft")!
     case .some("WordPress"): return UIImage(named: "WordPress")!
-    case .some("www.fastmail.com"): return UIImage(named: "FastMail")!
```

Comment:
This entry was added in addition to the `FastMail` one because the QR-codes Fastmail provided set issuer to `www.fastmail.com`. Without it, scanning the QR-codes would not make the icon visible.

I tested this again and Fastmail has since changed the issuer to `Fastmail` with a lowercase m. Therefore, I think we should update the `FastMail` entry to say `Fastmail` in addition to deleting this one.

Reply:
Done. Perhaps a case-insensitive string comparison could be useful for situations like these.